airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/WorkspaceHelper.java

```diff
@@ -37,14 +37,16 @@
 import io.airbyte.config.persistence.ConfigNotFoundException;
 import io.airbyte.config.persistence.ConfigRepository;
 import io.airbyte.scheduler.models.Job;
-import io.airbyte.scheduler.persistence.JobPersistence;
 import io.airbyte.validation.json.JsonValidationException;
 import java.io.IOException;
 import java.util.Objects;
 import java.util.UUID;
 import java.util.concurrent.ExecutionException;
 import org.checkerframework.checker.nullness.qual.NonNull;
 
+// todo (cgardens) - this class is in an unintuitive module. it is weird that you need to import
```

Comment:
I had to move this class so that `JobNotifier` and `JobTracker` could get at it. feels like a lame brain move, but I couldn't find a better place, and was reticent to add another module just for this.